app/src/main/res/values/strings.xml

```diff
@@ -711,5 +711,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="depicts_step_title">Depicts</string>
   <string name="license_step_title">Media License</string>
   <string name="media_detail_step_title">Media Details</string>
+  <string name="menu_view_category_page">View category page</string>
+  <string name="view_item_page">View item page</string>
```

Comment:
What about renaming "view_item_page" to "menu_view_item_page", just to be consistent with the string above ("menu_view_category_page") ?